Vagrantfile

```diff
@@ -67,11 +78,30 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     override.vm.synced_folder '.', '/home/vagrant/sync', disabled: true
   end
 
+  # OpenStack VMs
+  config.vm.provider :openstack do |os|
+    config.vm.synced_folder ".", "/home/#{USER}/vagrant", disabled: true
+    config.ssh.username = USER
+    config.ssh.private_key_path = settings['os_ssh_private_key_path']
+    config.ssh.pty = true
+    os.openstack_auth_url = settings['os_openstack_auth_url']
+    os.username = settings['os_username']
```

Comment:
make it also able to accept ENV, like
`os.username     = ENV['OS_USERNAME']
os.password     = ENV['OS_PASSWORD']`